MOBILE_CLIENT/COMMON/_HIGH/OAUTH_SCHEME_ACCOUNT_TAKEOVER/description.md

```diff
@@ -0,0 +1,149 @@
+The vulnerability arises from the application's use of a custom scheme in the `redirect_uri` parameter during OAuth authentication. 
+
+In a typical OAuth scenario, `redirect_uri` should be guaranteed to belong to the client application (identified by `client_id`) that requests data from an identity provider (Google, Facebook, Github...), using a custom scheme breaks that premise as it can be claimed by application on the user device.
```

Comment:
```suggestion
In a typical OAuth scenario, `redirect_uri` should be guaranteed to belong to the client application (identified by `client_id`) that requests data from an identity provider (Google, Facebook, Github...). Using a custom scheme breaks that premise as it can be claimed by the application on the user's device.
```